be/src/Project/Services/Catalog/Catalog.Api/v1/Categories/Controllers/CategoriesController.cs

```diff
@@ -388,5 +386,50 @@ public async Task<IActionResult> GetCategorySchemaByCategoryId(Guid? categoryId)
 
             throw new CustomException(string.Join(ErrorConstants.ErrorMessagesSeparator, validationResult.Errors.Select(x => x.ErrorMessage)), (int)HttpStatusCode.UnprocessableEntity);
         }
+
+        /// <summary>
+        /// Gets schemas by category id.
+        /// </summary>
+        /// <param name="categoryId">The category id.</param>
+        /// <returns>The category schema.</returns>
+        [HttpGet, MapToApiVersion("1.0")]
+        [Route("CategorySchemas/{categoryId}")]
+        [AllowAnonymous]
```

Comment:
**suggestion:** Endpoint has missing properties and response model should look something like:

![obraz](https://github.com/dawiddworak88/Giuru/assets/93117743/2b876642-bd23-4153-9f6a-c262bb27485d)